service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -1,5 +1,5 @@
 :_content-type: ASSEMBLY
-[id="upgrading-ossm"]
+[id="upgrading-service-mesh"]
```

Comment:
I confirmed with Kathryn that the anchor ID needs to match the file name. While this link to the docs guidelines is specific to modules, Kathryn confirmed it applies to assemblies: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Changing the file name would require updating the docs in around 13 places, plus `git mv` to track the file name change correctly.

Or change the id= back to `upgrading-ossm`.